app/javascript/src/lib/OWLanguageLinter.ts

```diff
@@ -315,8 +315,10 @@ function checkTranslations(content: string): void {
     }
 
     if (translateArguments.length > 1 && isStatic) {
+      const secondArgumentStart = translateArgumentsString.indexOf(translateArguments[1])
```

Comment:
Smart! One little suggestion though

```suggestion
      const secondArgumentStart = translateArgumentsString.indexOf(translateArguments[1], translateArguments[0].length)
```

Make the search start from the end of the first argument. Just in case the first argument is the same as the second one or something lol.